torchbenchmark/util/model.py

```diff
@@ -193,7 +193,7 @@ def _determine_batch_size(self, user_specified_batch_size=None):
 
```

Comment:
could we also update the comment here:
```
# batch size priority for eval tests: not ALLOW_CUSTOMIZE_BSIZE > user specified > default > device specified
```

this also gets me wondering, when is it intended to use device specified batch size @xuzhao9? I think all models specify their defaults

Reply:
I find this a little difficult to interpret `not ALLOW_CUSTOMIZE_BSIZE > user specified > default > device specified` already. What does it mean if `ALLOW_CUSTOMIZE_BSIZE == True` ?

Reply:
Maybe the code is sufficient ?

Reply:
I interpreted the "not ALLOW_CUSTOMIZE_BSIZE" as a condition check, it does seem that adding expanding it would end up being pseudo code